sources/packages/backend/apps/api/src/route-controllers/designation-agreement/models/designation-agreement.model.ts

```diff
@@ -55,22 +56,12 @@ export interface PendingDesignationDto extends GetDesignationAgreementsDto {
   legalOperatingName: string;
 }
 
-export interface UpdateDesignationLocation {
+export class UpdateDesignationLocation {
```

Comment:
I created this tech debt to reorganize the Designation Agreement and check for others.
https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1495